mesonbuild/compilers/asm.py

```diff
@@ -157,7 +157,8 @@ class MasmCompiler(Compiler):
     def get_compile_only_args(self) -> T.List[str]:
         return ['/c']
 
-    def get_argument_syntax(self) -> str:
+    @staticmethod
```

Comment:
This addition of `@staticmethod` seems unrelated to the fix. I think it should go in a separate commit.

Reply:
done